File: java/83_Remove_Duplicates_from_Sorted_List.java
class Solution {
    public ListNode deleteDuplicates(ListNode head) {
        ListNode ptr = head;
        while(ptr != null && ptr.next != null){
            if(ptr.next.val == ptr.val){
                ptr.next = ptr.next.next;
            }
            else{
                ptr = ptr.next;
            }
        }
        return head;
    }
}

File: python/83_Remove_Duplicates_from_Sorted_List.py
# Because it's sorted list, so just go straight forward
class Solution:
    def deleteDuplicates(self, head: ListNode) -> ListNode:
        ptr = head
        while ptr and ptr.next:
            if ptr.val == ptr.next.val:
                ptr.next = ptr.next.next
            else:
                ptr = ptr.next
        return head
